age--1.0.0--1.1.0.sql

```diff
@@ -0,0 +1,247 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
is there 1.0.0 version for PG12 ? Should this update file be added ?

Reply:
Removed.